studio/pages/project/[ref]/database/hooks/index.tsx

```diff
@@ -33,24 +30,23 @@ const HooksPage: NextPageWithLayout = () => {
   const canReadWebhooks = useCheckPermissions(PermissionAction.TENANT_SQL_ADMIN_READ, 'triggers')
   const canCreateWebhooks = useCheckPermissions(PermissionAction.TENANT_SQL_ADMIN_WRITE, 'triggers')
 
-  useEffect(() => {
-    if (ui.selectedProjectRef) meta.hooks.load()
-  }, [ui.selectedProjectRef])
-
-  const enableHooksForProject = async () => {
-    const res = await post(`${API_URL}/database/${ref}/hook-enable`, {})
-    if (res.error) {
-      ui.setNotification({
-        category: 'error',
-        message: `Failed to enable webhooks: ${res.error.message}`,
-      })
-    } else {
+  const { mutate: enableHooks } = useHooksEnableMutation({
```

Comment:
When clicking the button for enable webhooks, it doesn't show a spinner – would you mind adding one? 🙏🏻

Reply:
gotcha just pushed! 🙏